app/controllers/spina/journal/primer_theme/application_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Spina
+  module Journal
+    module PrimerTheme
+      # Base class from which controllers related to the journal plugin inherit
+      class ApplicationController < Spina::ApplicationController
```

Comment:
I think there could be advantages of putting stuff under `Spina::Conferences::PrimerTheme::Journal`, especially since the stuff in this gem is all under `Spina::Conferences::PrimerTheme`—though if you'd prefer you could also release a separate frontend gem and then we could add both of those to the main app

Reply:
See my PR comment. Out of these two options I think a separate gem makes more sense, assuming implementing this is reasonably trivial.